src/pages/Jobs.js

```diff
@@ -22,11 +22,15 @@ export default function Jobs() {
     let url =`${process.env.REACT_APP_BACKEND_SERVER_URL}/jobs`;
     let data = await fetch(url);
     let result = await data.json();
+    console.log("rr",result)
     setOriginalJobs(result);
+    //setJobs(result)
   };
 
   const handleSearch = (e) => {
     let filteredJobs = originalJobs;
+   //let filteredJobs = [];
+    console.log("hehe",keyword)
     if (e) {
```

Comment:
add a quick comment here explaining why we check for e (to differentiate between landing on this page, versus searching)